features/background_requires_scenario.feature

```diff
@@ -53,5 +54,5 @@ Feature: Background Requires Scenario
     When I run `ruby lint.rb`
     Then it should pass with exactly:
       """
-
+      There are no issues
```

Comment:
this makes Tests fragile. everytime changing this Output, would Need to Change every test.
could we add a --verbose option, which issues such a message or a --quiet which suppresses such a message. Me for example do not want to read it. I trust the return code.